controller/users_blackbox_test.go

```diff
@@ -233,7 +238,7 @@ func (s *TestUsersSuite) TestUpdateUserUnsetVariableInContextInfo() {
 		"count":        3,
 	}
 
-	updateUsersPayload = createUpdateUsersPayload(&newEmail, &newFullName, &newBio, &newImageURL, &newProfileURL, contextInformation)
```

Comment:
Thanks @hectorj2f for the review:
I am slightly confused -
Do you mean to say, why did I pass `nil` for `company *string` instead of passing something real?
I wanted to keep it optional since those tests weren't directly about testing the `company`. I've put the `company` in the `TestUpdateUserOK` . No strong feelings about this though - feel free to tell me if I need to put the `company` in all other tests as well :)